react-native-gutenberg-bridge/ios/GutenbergBridgeDelegate.swift

```diff
@@ -49,6 +57,10 @@ public protocol GutenbergBridgeDelegate: class {
     /// Tells the delegate that the editor view has completed the initial render.
     ///
     func gutenbergDidMount(hasUnsupportedBlocks: Bool)
+
+    /// Tells the delegate that logger method is called.
+    ///
+    func nativeLoggingHook(message: String, logLevel: LogLevel)
```

Comment:
Could we update the name to be consistent with delegate method naming on iOS? some alternatives might be:
gutenbergDidLog(message: String, logLevel: LogLevel) 
gutenbergDidGenerateLog(message: String, logLevel: LogLevel)

Reply:
Thx! 🙇 51183cf